### DISCUSSION TITLE
Change font size and font name

### DISCUSSION THREAD

**Discussion by lhoupert** [mmistakes/minimal-mistakes#2592] [Q&A] [answered]
Hi! Thank you for this excellent theme! 

I am new in using jekyll-theme and I had a couple of basic questions about the webpages I am creating here: https://lhoupert.github.io/ (the git repo associated is  https://github.com/lhoupert/lhoupert.github.io)

1) how can I change the font size and font name for all pages ? I would like to use the same font style than http://madebymunsters.github.io/Lannister/

2) Is it easy to change the font size and name only for a specific webpage? I would like to have smaller font for a bibliography section

Thank you very much

Loíc

**Comment by mmistakes** [comment, marked as answer]
Check the pinned and closed issues:

- https://github.com/mmistakes/minimal-mistakes/issues/1352
- https://github.com/mmistakes/minimal-mistakes/issues/2435
- https://github.com/mmistakes/minimal-mistakes/issues/2290
- https://github.com/mmistakes/minimal-mistakes/issues/1219

**Comment by lhoupert** [comment]
Thank you very much for your quick answer it is working very well.
